config/carbonapi.yaml

```diff
@@ -35,23 +37,16 @@ tz: ""
 # to get reasonable performance
 #
 # For go-carbon --- it depends on how you use it.
-sendGlobsAsIs: false
+sendGlobsAsIs: true
 # If sendGlobsAsIs is set and resulting response will be larger than maxBatchSize
 # carbonapi will revert to old behavir. This allows you to use benifits of passing
 # globs as is but keep memory usage in sane limits.
 #
 # For go-carbon you might want it to keep in some reasonable limits, 100 is good "safe" defaults
 #
 # For some backends (e.x. graphite-clickhouse) you might want to set it to some insanly high value, like 100000
-
-# If 'true', carbonapi will send requests as is, with globs and braces
-# even without checking maxBatchSize
-# in result it can increase performance for graphite-clickhouse by removing /metric/find request before calling /render
-alwaysSendGlobsAsIs: false
-
-functionsConfigs:
-    graphiteWeb: ./graphiteWeb.example.yaml
```

Comment:
We need to keep config parameters even if we don't use them, this increases visibility for those who can't read the code about available options. You can comment it out instead

Reply:
I have added these back as comments.